sentry-kotlin-multiplatform/src/commonMain/kotlin/io/sentry/kotlin/multiplatform/SentryBreadcrumbFactory.kt

```diff
@@ -0,0 +1,111 @@
+package io.sentry.kotlin.multiplatform
+
+/**
+ * The factory is responsible for creating Breadcrumb objects in SentryBreadcrumb
+ * in order to avoid code duplication.
+ *
+ * This is only for internal usage and is not exposed to the user.
+ */
+internal object SentryBreadcrumbFactory {
```

Comment:
Wouldn't be possible to just call the Breadcrumb class from Native and convert it back to the KMP type?